app/templates/people/person/personal-information/index.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuToolbar>
+        <AuToolbarGroup>
+          <AuHeading>
+            Persoonlijke gegevens
+          </AuHeading>
+        </AuToolbarGroup>
+        <AuToolbarGroup>
+          <AuLink
+            @route="people.person.personal-information.edit"
+            @skin="button-secondary"
+            @icon="pencil"
+            @iconAlignment="left"
+          >
+            Bewerk
+          </AuLink>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuHeading @level="2" @skin="3">
+        {{@model.person.givenName}} {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Voornaam</:label>
+            <:content>{{@model.person.givenName}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
```

Comment:
These can all be added under the same `<Grid>` component. A new `<Grid>` is only needed if you want to force things on a new line.

Reply:
This is the design, no? Because the second column will be the date of birth and RRN, no?

Reply:
I assumed the items would just flow through to the next line if needed. Forcing items on a new line because it's there in the design isn't the best reason I think. The other data isn't there yet so in my head the last name just moves to the first row.

Reply:
Yes, but they're expected to be on the left side; having things moving depending on other attributes is not consistent on all pages.

Reply:
Well it's only my opinion, showing them in new row while the right column is empty looks weird to me. Anyway, leave it as is then.